defaults/krill.conf

```diff
@@ -244,6 +244,15 @@
 #
 ### post_limit_rfc6492 = 1048576
 
+# Set the timeout for RFC 6492 and RFC 8181 client http connections to the parent
```

Comment:
s/http/HTTP/

Also, to be clear this does NOT affect the time permitted for an initial connection to be established, rather once connected it (quoting the [reqwest docs](https://docs.rs/reqwest/0.11.5/reqwest/struct.ClientBuilder.html#method.timeout)) is the maximum time allowed "from when the request starts connecting until the response body has finished".

It's a bit wordy but one alternative could be "Set the timeout for a complete RFC 6492 and RFC 8181 client HTTP request-response round-trip to the parent or publisher, excluding the time required to establish the connection." ?

Reply:
fixed